AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_PyTorch_GettingStarted/README.md

```diff
@@ -121,26 +124,6 @@ The sample uses pretrained model provided by Intel and published as part of [Int
 
 If you receive an error message, troubleshoot the problem using the **Diagnostics Utility for Intel® oneAPI Toolkits**. The diagnostic utility provides configuration and system checks to help find missing dependencies, permissions errors, and other issues. See the *[Diagnostics Utility for Intel® oneAPI Toolkits User Guide](https://www.intel.com/content/www/us/en/develop/documentation/diagnostic-utility-user-guide/top.html)* for more information on using the utility.

```

Comment:
@louie-tsai Need to change title case to ### Run the Jupyter Notebook.

Reply:
addressed it accordingly.